packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -274,6 +279,21 @@ export class GcdsSelect {
     }
 
     this.inheritedAttributes = inheritAttributes(this.el, this.shadowElement);
+
+    if (this.el.children) {
+      this.options = Array.from(this.el.children);
+      this.options.map(option => {
```

Comment:
Since you are not mapping to a new array, using `forEach` here would be a bit more efficient.

Reply:
Additionally, you can reduce some code repetition by creating a variable for `.getAttribute('value')`:
`const value = option.getAttribute('value');`